x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -179,9 +179,9 @@ The following is a third portion of the same `005010X222A2` `837` transaction se
 
 These two definitions apply to different instances of the `DTP` segment, as part of the 2300 Claim Information loop. In the transaction set structure these uses of the `DTP` segment occur essentially in the same position, matching two possible occurrences of a repeating DTP segment in the base X12 standard. Because the two uses of the segment are supplying different information, the HIPAA standard gives them different names and interprets the `DTP03` field in different ways.
 
-In this case the data value in the first field of the segment, the `Date Time Qualifier` field, identifies which variation of the segment is actually being used. Since the code set for this field has different values for each of these uses the value present in the field tells whether the DTP segment in a parsed document is a `DTP_DateAccident` or a `DTP_DateAcuteManifestation` (or any of several other uses of the DTP segment in the same position). The `varValue: true` flag in the schema definition indicates that this first field is used in this way to distinguish between variations.
+In this case the data value in the first field of the segment, the `Date Time Qualifier` field, identifies which variation of the segment is actually being used. Since the code set for this field has different values for each of these uses the value present in the field tells whether the DTP segment in a parsed document is a `DTP_DateAccident` or a `DTP_DateAcuteManifestation` (or any of several other uses of the DTP segment in the same position). The `varValue: true` flag in the schema definition indicates that this first field is used in this manner to distinguish between variations.
```

Comment:
```suggestion
In this case, the data value in the first field of the segment, the `Date Time Qualifier` field, identifies which variation of the segment is actually being used. Since the code set for this field has different values for each of these uses, the value present in the field tells whether the DTP segment in a parsed document is a `DTP_DateAccident` or a `DTP_DateAcuteManifestation` (or any of several other uses of the DTP segment in the same position). The `varValue: true` flag in the schema definition indicates that this first field is used in this manner to distinguish between variations.
```